app/components/Story.js

```diff
@@ -85,13 +91,15 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   button: {
-    paddingTop: 10,
-    paddingHorizontal: 10,
-    marginTop: 15,
+
   },
   toggle: {
-    paddingVertical: 10,
-    paddingHorizontal: 10,
+
+  },
+  bar: {
+    position: "absolute",
```

Comment:
not a huge deal, but there's a mixture of single/double quotes here. We don't have concrete style guide, but the convention a lot of react projects use is double quotes for react properties (e.g. `<Icon fill="crimson" size="25" name="hamburger" />`) and single quotes everywhere else (e.g. position: 'absolute')